modules/isolation_segment/subnets.tf

```diff
@@ -0,0 +1,9 @@
+resource "azurerm_subnet" "iso_seg_subnet" {
+  name  = "${var.environment}-iso-seg-subnet-${element(var.iso_seg_names, count.index)}"
+  count = "${var.count}"
+
+  resource_group_name       = "${var.resource_group_name}"
+  virtual_network_name      = "${var.network_name}"
+  address_prefix            = "${element(var.iso_seg_subnet_blocks, count.index)}"
+  network_security_group_id = "${var.bosh_deployed_vms_security_group_id}"
```

Comment:
That was an oversight. I will remove that security group.

Reply:
@malston Does removing the security group still work? Because we believe this will not allow the VMs deployed to the Isolation Segment to talk to the bosh director.

cc @Polar-Beard